system_metrics_collector/test/system_metrics_collector/test_linux_process_cpu_measurement_node.cpp

```diff
@@ -77,147 +88,12 @@ class MockLinuxProcessCpuMeasurementNode : public system_metrics_collector::
 private:
   system_metrics_collector::ProcPidCpuData MakeSingleMeasurement() override
   {
-    EXPECT_GT(test_constants::kProcPidSamples.size(), measurement_index_);
-    return test_constants::kProcPidSamples[measurement_index_++];
+    EXPECT_GT(kProcPidSamples.size(), measurement_index_);
```

Comment:
Should this be an `ASSERT` to make sure the next line never goes OOB?

Reply:
beat you to it by 7 minutes